src/loadimg/utils.py

```diff
@@ -104,10 +136,15 @@ def isBase64(sb):
     """
     try:
         if isinstance(sb, str):
-            # If there's any unicode here, an exception will be thrown and the function will return false
+            sb = re.sub(r"^data:image\/[a-zA-Z]+;base64,", "", sb)
             sb_bytes = bytes(sb, "ascii")
         elif isinstance(sb, bytes):
             sb_bytes = sb
         return base64.b64encode(base64.b64decode(sb_bytes)) == sb_bytes
     except Exception:
         return False
+
+
+if __name__ == "__main__":
+    img_str = """data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABYAAAAWCAYAAADEtGw7AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAwSURBVEhLY7SxsfnPQAPABKWpDkYNhoNRg+Fg1GA4GDUYDkYNhoNRg+FgqBnMwAAAqlsB32Fk5YcAAAAASUVORK5CYII="""
+    print(load_img(img=img_str, output_type="str"))
```

Comment:
thanks for the example code, but I think we should remove this from production, I am thinking of adding tests soon to this repo, once that's done we can use this.